### PR TITLE
Something from opensky api

### LINKED ISSUE #406: Something from opensky api
- [ ] Define what are parameter and result 
- [ ] Opensky library
- [ ] Logic script to store in chain

### PR DESCRIPTION
**Pull request by prin-r**
Fix #406

### CODE REVIEW

File: owasm/src/ext/flight/openskynetwork.rs [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
...  Flight::new("a79de9",&FlightOption::DEPARTURE, "EDDF", 1580937440,1580997440).from_cmd_output(r#"[{"icao24":"406459","firstSeen":1580938171,"estDepartureAirport":"EDDF","lastSeen":1580941248,"estArrivalAirport":"EDDP","callsign":"BCS30C ","estDepartureAirportHorizDistance":2588,"estDepartureAirportVertDistance":194,"estArrivalAirportHorizDistance":7435,"estArrivalAirportVertDistance":132,"departureAirportCandidatesCount":1,"arrivalAirportCandidatesCount":6},{"icao24":"a79de9","firstSeen":1580937606,"estDepartureAirport":"EDDF","lastSeen":1580941006,"estArrivalAirport":"LFPG","callsign":"FDX36","estDepartureAirportHorizDistance":2854,"estDepartureAirportVertDistance":187,"estArrivalAirportHorizDistance":2411,"estArrivalAirportVertDistance":157,"departureAirportCandidatesCount":2,"arrivalAirportCandidatesCount":8},{"icao24":"3c5426","firstSeen":1580937574,"estDepartureAirport":"EDDF","lastSeen":1580940348,"estArrivalAirport":"EDDP","callsign":"BCS27H ","estDepartureAirportHorizDistance":2669,"estDepartureAirportVertDistance":194,"estArrivalAirportHorizDistance":4686,"estArrivalAirportVertDistance":25,"departureAirportCandidatesCount":2,"arrivalAirportCandidatesCount":6}]"#.into()),
```
Comment (taobun) [inline owasm/src/ext/flight/openskynetwork.rs]: Auto format not work?

File: owasm/src/ext/flight/openskynetwork.rs [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+            Some(true)
+        );
+    }
+
+    #[test]
+    fn test_from_cmd_not_found_flight_ok() {
+        assert_eq!(
+            Flight::new("a79de9",&FlightOption::DEPARTURE, "EDDF", 1580937440,1580997440).from_cmd_output(r#"[{"icao24":"406459","firstSeen":1580938171,"estDepartureAirport":"EDDF","lastSeen":1580941248,"estArrivalAirport":"EDDP","callsign":"BCS30C ","estDepartureAirportHorizDistance":2588,"estDepartureAirportVertDistance":194,"estArrivalAirportHorizDistance":7435,"estArrivalAirportVertDistance":132,"departureAirportCandidatesCount":1,"arrivalAirportCandidatesCount":6},{"icao24":"3c5426","firstSeen":1580937574,"estDepartureAirport":"EDDF","lastSeen":1580940348,"estArrivalAirport":"EDDP","callsign":"BCS27H ","estDepartureAirportHorizDistance":2669,"estDepartureAirportVertDistance":194,"estArrivalAirportHorizDistance":4686,"estArrivalAirportVertDistance":25,"departureAirportCandidatesCount":2,"arrivalAirportCandidatesCount":6}]"#.into()),
```
Comment (taobun) [inline owasm/src/ext/flight/openskynetwork.rs]: Between 2 number and comma why not have whitespace?

**Review by taobun — CHANGES_REQUESTED** [review]

File: owasm/src/ext/flight/openskynetwork.rs (hunk 1 above)
Comment (prin-r) [reply owasm/src/ext/flight/openskynetwork.rs]: done

File: owasm/src/ext/flight/openskynetwork.rs (hunk 2 above)
Comment (prin-r) [reply owasm/src/ext/flight/openskynetwork.rs]: done

**Review by taobun — APPROVED** [review]

**Review by sorawit — APPROVED** [review]

**Pull request merged**